lib/js_routes.rb

```diff
@@ -19,7 +19,8 @@ class JsRoutes
     url_links: nil,
     camel_case: false,
     default_url_options: {},
-    compact: false
+    compact: false,
+    serializer: "this.default_serializer"
```

Comment:
@bogdan I'm not totally sure I follow. How does this work with the configurable namespace? We can't set the default to a public method on `JsRoutes` because we don't know that `JsRoutes` exists.

It might be better to just make `default_serializer` public when we run `createGlobalJsRoutesObject`, and then just add a README note that it is public on the namespace that the user defines, rather than try to get the default value to include the name of the configurable namespace.